src/commands/init.ts

```diff
@@ -23,7 +24,7 @@ export class InitCommand implements Command {
   run(options): Promise<any> {
     return new Promise((resolve, reject) => {
       let env = new YeomanEnvironment();
-      env.register(require.resolve('generator-polymer-init'), 'polymer-init:app');
+      env.registerStub(polymerGenerator, 'polymer-init:app');
```

Comment:
`require.resolve()` was giving us trouble with `depcheck`, since it technically was never required in our application. Instead of asking Yeoman to load the generator for us, we can require it ourselves.